images/arch/Containerfile

```diff
@@ -12,6 +12,12 @@ COPY extra-packages /
 RUN pacman -Syu --needed --noconfirm - < extra-packages
 RUN rm /extra-packages
 
+# Enable man pages, enable progress bars, enable colors, enable up to 15 parallel downloads
+RUN sed -i -e 's/NoProgressBar/#NoProgressBar/' -e 's/#Color/Color/' -e 's/NoExtract/#NoExtract/' /etc/pacman.conf
```

Comment:
This no longer enables ParallelDownloads, and enabling color / progressbars seem like a rather personal change. I rather keep it stock.

Reply:
The `NoProgressBar` change is changed to the stock option. fwiw